packages/stylelint-config-nimble/tests/test.css

```diff
@@ -0,0 +1,348 @@
+/* color-hex-case */
+a { color: #fff; }
+
+/* color-hex-length */
+a { color: #ffffff; }
+
+/* color-named */
+a { color: black; }
+
+/* color-no-invalid-hex */
+a { color: #00; }
+
+a {
+top: 0px;
+}
+
+/* font-family-name-quotes */
+a { font-family: Times New Roman, Times, serif; }
```

Comment:
[joke] 
  Should we add a stylelint rule to forbide `Times New Roman`? :harold: 
[/joke]